DISPLAY "(null)" invalid; disabling X11 forwarding
INFO:tensorflow:PS hosts are: ['raiders1:2200']
INFO:tensorflow:Worker hosts are: ['raiders1:2201', 'raiders1:2202', 'raiders1:2203', 'raiders1:2204', 'raiders2:2205', 'raiders2:2206', 'raiders2:2207', 'raiders2:2208', 'raiders3:2209', 'raiders3:2210', 'raiders3:2211', 'raiders3:2212', 'raiders8:2213', 'raiders8:2214', 'raiders8:2215', 'raiders8:2216']
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job ps -> {raiders1:2200}
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job worker -> {raiders1:2201, raiders1:2202, raiders1:2203, raiders1:2204, raiders2:2205, raiders2:2206, raiders2:2207, raiders2:2208, raiders3:2209, raiders3:2210, raiders3:2211, raiders3:2212, raiders8:2213, localhost:2214, raiders8:2215, raiders8:2216}
I tensorflow/core/distributed_runtime/rpc/grpc_server_lib.cc:202] Started server with target: grpc://localhost:2214
INFO:tensorflow:Learning rate: 0.005000, momentum: 0.600000
INFO:tensorflow:Sync mode!!!!!!
INFO:tensorflow:Compute groups : 1
INFO:tensorflow:SyncReplicas enabled: replicas_to_aggregate=16; total_num_replicas=16
INFO:tensorflow:Num nodes per compute groups: 16
INFO:tensorflow:Tokens per step: 16
INFO:tensorflow:2016-07-26 07:50:15.401692 Supervisor
E0726 07:50:27.305432368   39367 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.45:2209': timeout occurred
E0726 07:50:27.305512931   39367 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.45:2210': timeout occurred
E0726 07:50:27.305533951   39367 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.45:2211': timeout occurred
E0726 07:50:27.305553471   39367 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.45:2212': timeout occurred
E0726 07:50:27.458234758   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 07:50:27.458323050   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 07:50:27.458348340   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 07:50:27.458368287   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 07:50:30.458050347   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 07:50:30.458294403   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 07:50:30.458328768   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 07:50:30.458354074   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 07:50:36.482052543   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 07:50:36.482118448   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 07:50:36.482142756   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 07:50:36.482265097   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 07:50:39.482059221   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 07:50:42.482035520   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 07:50:42.482090640   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 07:50:45.482012939   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 07:50:51.481983811   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 07:50:51.482032843   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 07:50:54.482031897   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 07:51:00.481946917   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 07:51:07.241991773   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 07:51:07.242048744   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 07:51:16.241990648   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 07:51:25.337953608   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 07:51:33.689972115   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 07:51:33.690016236   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 07:51:39.689924847   39108 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
run-expr-32node.sh: line 23: 38972 Terminated              CUDA_VISIBLE_DEVICES='' bazel-bin/inception/imagenet_distributed_train --batch_size=16 --data_dir=/lfs/local/0/daniter/16node-data-$2 --job_name='worker' --task_id=$2 --ps_hosts='raiders1:2200' --worker_hosts='raiders1:2201,raiders1:2202,raiders1:2203,raiders1:2204,raiders2:2205,raiders2:2206,raiders2:2207,raiders2:2208,raiders3:2209,raiders3:2210,raiders3:2211,raiders3:2212,raiders8:2213,raiders8:2214,raiders8:2215,raiders8:2216' --initial_learning_rate=$3 --momentum=$4 --sync=True --compute_groups=$5
